src/cargo/util/toml/targets.rs

```diff
@@ -105,7 +105,10 @@ pub(super) fn to_targets(
         if metabuild.is_some() {
             anyhow::bail!("cannot specify both `metabuild` and `build`");
         }
-        let custom_build = Path::new(custom_build);
+        if custom_build.len() > 1 {
+            anyhow::bail!("multiple build scripts feature is not implemented yet! ")
+        }
+        let custom_build = Path::new(&custom_build[0]);
         let name = format!(
             "build-script-{}",
             custom_build
```

Comment:
1. We'll need to loop over
2. Do we need to make build script names more unique?